internal/acceptance/lakeview_dashboard_test.go

```diff
@@ -0,0 +1,481 @@
+		Check: resourceCheck("databricks_lakeview_dashboard.d1", func(ctx context.Context, client *common.DatabricksClient, id string) error {
+			w, err := client.WorkspaceClient()
+			if err != nil {
+				return err
+			}
+			dashboard, err := w.Lakeview.Get(ctx, dashboards.GetDashboardRequest{
+				DashboardId: id,
+			})
+			if err != nil {
+				return err
+			}
+			assert.Equal(t, "Monthly Traffic Report", dashboard.DisplayName)
+			fmt.Println(dashboard.SerializedDashboard)
+			require.NoError(t, err)
+			require.Equal(t, dashboard.UpdateTime, initial_update_time)
+			return nil
+		}),
+	})
+}
+
+func TestAccDashboardWithRemoteChange(t *testing.T) {
+	dashboard_id := ""
+	display_name := ""
+	warehouse_id := ""
+	etag := ""
+	var template TemplateStruct
+	workspaceLevel(t, step{
+		Template: MakeTemplate(template.SetAttributes(map[string]string{
+			"display_name":         "Monthly Traffic Report",
+			"warehouse_id":         "{env.TEST_DEFAULT_WAREHOUSE_ID}",
+			"parent_path":          "/Shared/provider-test",
+			"serialized_dashboard": `{\"pages\":[{\"name\":\"new_name\",\"displayName\":\"New Page\"}]}`,
+		})),
+		Check: resourceCheck("databricks_lakeview_dashboard.d1", func(ctx context.Context, client *common.DatabricksClient, id string) error {
+			w, err := client.WorkspaceClient()
+			if err != nil {
+				return err
+			}
+			dashboard, err := w.Lakeview.Get(ctx, dashboards.GetDashboardRequest{
+				DashboardId: id,
+			})
+			if err != nil {
+				return err
+			}
+			assert.Equal(t, "Monthly Traffic Report", dashboard.DisplayName)
+			fmt.Println(dashboard.SerializedDashboard)
+			require.NoError(t, err)
+			dashboard_id = dashboard.DashboardId
+			display_name = dashboard.DisplayName
+			warehouse_id = dashboard.WarehouseId
+			etag = dashboard.Etag
+			return nil
+		}),
+	}, step{
+		PreConfig: func() {
+			w, err := databricks.NewWorkspaceClient(&databricks.Config{})
+			if err != nil {
+				fmt.Println(err)
+			}
+			_, err = w.Lakeview.Update(context.Background(), dashboards.UpdateDashboardRequest{
+				DashboardId:         dashboard_id,
+				DisplayName:         display_name,
+				Etag:                etag,
+				WarehouseId:         warehouse_id,
+				SerializedDashboard: "{\"pages\":[{\"name\":\"b532570b\",\"displayName\":\"New Page Modified Remote\"}]}",
+			})
+			if err != nil {
+				fmt.Println(err)
+			}
+		},
+		Template: MakeTemplate(template),
+		Check: resourceCheck("databricks_lakeview_dashboard.d1", func(ctx context.Context, client *common.DatabricksClient, id string) error {
+			w, err := client.WorkspaceClient()
+			if err != nil {
+				return err
+			}
+			dashboard, err := w.Lakeview.Get(ctx, dashboards.GetDashboardRequest{
+				DashboardId: id,
+			})
+			if err != nil {
+				return err
+			}
+			assert.Equal(t, "Monthly Traffic Report", dashboard.DisplayName)
+			fmt.Println(dashboard.SerializedDashboard)
+			require.NoError(t, err)
+			require.NotEqual(t, dashboard.UpdateTime, dashboard.CreateTime)
```

Comment:
You also should check that it has been updated by tf. This `UpdateTime` could be the one by your manual modification.

Reply:
So basically also add the check to compare update time from GET and from the local terraform state?

Reply:
Either time, or the etag.